### DISCUSSION TITLE
Board - More detailed Rules section

### DISCUSSION THREAD

**Discussion by BoldBigflank** [BattlesnakeOfficial/feedback#138] [Feature Requests]
Our casters sometimes wonder how much damage the hazard sauce does, but have no way of seeing it on the page. I suggest adding this to the main page, like so:

![image](https://user-images.githubusercontent.com/636964/142752856-4e08b956-b30d-4529-b5b3-888e446d2cc4.png)

There might be other ways of doing it, like hiding it inside a collapsed div. I think that if there's space for it, it'd be nice for viewers to be able to see it whenever they want. Perhaps there's a way to change the rule names to something more human readable too.

**Comment by bvanvugt** [comment]
@BoldBigflank this is a great suggestion! We're actually already working on something to solve this, should go live in the next few weeks.
